### DISCUSSION TITLE
Possibility to download a pdf of the guide https://quarto.org/docs/guide/

### DISCUSSION THREAD

**Discussion by Janidai** [quarto-dev/quarto-cli#5795] [Feature Requests]
### Description

Is it possible to have also a pdf of the [guide](https://quarto.org/docs/guide/).
with version like [python](https://docs.python.org/3.11/download.html)

**Comment by mcanouil** [comment]
Duplicate of #5412

There is currently no way to turn a website project into PDF, contrary to books.
